Add documentation for account search queries.

jwir3 opened the pull request.

shadowhand commented:
@jwir3 I just finished this in #69.

jwir3 commented:
Sure thing.

shadowhand commented on source/methods/accounts.md.erb:
````diff
@@ -70,6 +70,48 @@ There are a few other features that may be explicitly enabled on some accounts:
+> Example Request
+
+```shell
+curl <%=@api_prefix%>/2/login/find?q=123 \
````
I put this in the index, since it's technically a `login` endpoint not an `account` endpoint.

shadowhand commented on source/methods/accounts.md.erb:
```diff
@@ -70,6 +70,48 @@ There are a few other features that may be explicitly enabled on some accounts:
+
+<% json do %>
+[{
+  "account": <%= print_json(data.objects['account_search_result'], :include => {
```
I like this. Let's copy this onto the other PR.

jwir3 commented:
@shadowhand Done.

shadowhand commented on source/index.md.erb:
```diff
@@ -258,16 +258,20 @@ $result = $wiw->get("login/find?q=when");
-      "subdomain": "thisclicks",
-      "account_holder": "Jeff",
-      "logo": "https://s3.amazonaws.com/...",
-      "place": <%= print_json(data.objects['place']) %>,
```
We lost this. Can you add this back in for WIW and add `place: null` for the other?

jwir3 replied:
Sure, I'll add it back and re-push the branch.

jwir3 commented:
@shadowhand So I added back the data structure for places to the 'When I Work' result, but instead of adding an explicit `'place' => "null"` entry for that one result, I added a default `place: null` to the `Account` and `AccountSearchResult` objects.

shadowhand commented:
:shipit:
